js/src/tooltip.js

```diff
@@ -371,20 +369,21 @@ class Tooltip extends BaseComponent {
     element.innerHTML = this._config.template
 
     const tip = element.children[0]
+    this.setContent(tip)
     tip.classList.remove(CLASS_NAME_FADE, CLASS_NAME_SHOW)
 
     this.tip = tip
     return this.tip
   }
 
-  setContent() {
-    const tip = this.getTipElement()
+  setContent(tip) {
```

Comment:
Hm would it make sense to rename this to `this._setContent` or `this._setTipContent` even? I mean it's not meant to be used by users, is it?

Reply:
Yes you are extremely right on this. 

I plan to use it on **5.2** https://github.com/twbs/bootstrap/pull/34519 and provide it through documentation. 

I would prefer to leave it as it is for now, as it has some test over it, that I have changed on the next MR.

are we ok with this?